### ISSUE TITLE
Abnormal accuracy

### ISSUE THREAD

**Issue by xubin1994** [ztsrxh/RoadBEV#2]
I generated the ground truth using preprocess_gt.py and regenerated the left_half and right_half folders for the dataset,but got the metrics (stereo mode):
test:    abs_err:5.425, rmse:5.679, >0.5cm:99.95,
This accuracy seems to be an order of magnitude greater than what is described in the article.

**Comment by ztsrxh** [comment]
Thanks for your interest. We have provided the GT labels generated in our environment. You can use it thus eliminating potential problems in GT generation.  We found a mistake in _dataset.py_ last week, which has been corrected now. You can update this script.

**Comment by xubin1994** [comment]
I used the GT you provided, and the results I got were still the same as before。
I  visualize some of the results(RoadBEV/filenames/train/20230408031625.800.pkl). The image below is an example of the training set. The black part is to projected grid into the image part, ensuring that the input is correct.
![image](https://github.com/ztsrxh/RoadBEV/assets/28856641/2f56b773-f704-4f73-a0b8-395c53291386)
and the elevation result like that:
![image](https://github.com/ztsrxh/RoadBEV/assets/28856641/d68a73c0-661f-4652-8979-3c4663394c21)

**Comment by ztsrxh** [comment]
I cannot figure out why the visualized GT likes that. I visualize this sample, which looks like:
![myplot](https://github.com/ztsrxh/RoadBEV/assets/39629541/6d5363a4-2e08-479b-a749-cd60515fdeeb)


Something wrong with your code or environment.

**Comment by xubin1994** [comment]
gt same as you, the above sample is inference result

**Comment by ztsrxh** [comment]
have you updated the _dataset.py_?  How is your environment?

**Comment by xubin1994** [comment]
About the dataset.py:
I re-pulled the latest branch so there should be no problem.
About the environment:
I installed the same conda environment according to the readme.
The only thing that may be different is: the half-resolution image was generated by myself using opencv, because the downloaded RSRD data set does not have left_half and right_half folders

**Comment by ztsrxh** [comment]
I do not know where you download the dataset. You can download it again on our website.

**Comment by xubin1994** [comment]
I re-downloaded the dataset and the problem was solved.
But I seem to have found a problem with dataset.py， in https://github.com/ztsrxh/RoadBEV/blob/ca5bea1ef3a5e18749f6e2c34718e6d93eebcdd9/utils/dataset.py#L241, It seems like a deep copy should be added here，otherwise voxel_cam_right and voxel_cam_left will be the same, see the log below:
![img_v3_02a2_535c0c03-9efd-4f38-b379-e357495c231g](https://github.com/ztsrxh/RoadBEV/assets/28856641/f706e3e4-df66-4f7b-bf2a-fd80a81617b3)

**Comment by jiajigiser** [comment]
May I ask if it is possible to export requirements. txt (a list of all libraries and versions)? My CUDA version is 11.8, and I am not sure how to install the correct library version.

**Comment by ztsrxh** [comment]
> I re-downloaded the dataset and the problem was solved. But I seem to have found a problem with dataset.py， in
> 
> https://github.com/ztsrxh/RoadBEV/blob/ca5bea1ef3a5e18749f6e2c34718e6d93eebcdd9/utils/dataset.py#L241
> 
> , It seems like a deep copy should be added here，otherwise voxel_cam_right and voxel_cam_left will be the same, see the log below:
> ![image](https://private-user-images.githubusercontent.com/28856641/323555929-0fbb74d5-46a7-4b1f-980a-2cd6f5cc0698.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM0OTE4ODgsIm5iZiI6MTcxMzQ5MTU4OCwicGF0aCI6Ii8yODg1NjY0MS8zMjM1NTU5MjktMGZiYjc0ZDUtNDZhNy00YjFmLTk4MGEtMmNkNmY1Y2MwNjk4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE5VDAxNTMwOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQ2NTQ3MWNiODc2MmE4ZWQ3MDM3ZjI0NmQ0MGE5YzMzYTA5N2NkNWFlNWNkMjZlYmQyOTY2NjY0ZGJhZjNkYmYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.LjTsxPv7umDDjYSB2XfUMEHneQimhDjvtgZ1f9XPGdQ)

Thanks for your careful check. We have fixed this bug and now deepcopy the _voxel_cam_left_. The _ckpt_ weight for RoadBEV-stereo is also updated.

**Comment by jiajigiser** [comment]
May I ask how to visualize.pkl results(elevation result?) and how to view the results of 3D reconstruction?
